packages/react-conformance/src/types.ts

```diff
@@ -1,9 +1,13 @@
-export interface IsConformantOptions {
+import { ComponentDoc } from 'react-docgen-typescript';
```

Comment:
Let's add the pre-loaded component as an option, as well as the component path and whether to use the default export or a named export.
```ts
import * as React from 'react';
import { ComponentDoc } from 'react-docgen-typescript';

export interface TestingOptions<TProps = {}> {
  componentPath: string;
  Component: React.ComponentType<TProps>;
  constructorName: string;
  requiredProps?: Partial<TProps>;
  exportedAtTopLevel?: boolean;
  useDefaultExport?: boolean;
}
```

Then update the various tests that are `require`ing the component to use the pre-loaded one.